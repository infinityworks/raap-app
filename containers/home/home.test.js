/* eslint-env jest */

import React from 'react';
import * as renderer from 'react-test-renderer';
import Index from './_component';

const env = process.env.NODE_ENV || 'development';

if (env !== 'production') {
  const defaultProps = {
    createTodo: () => {},
    subscribeToNewTodos: () => {},
    todos: [],
  };

  describe('Home page', () => {
    it('matches the snapshot"', () => {
      const component = renderer.create(<Index {...defaultProps} />);
      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
}
