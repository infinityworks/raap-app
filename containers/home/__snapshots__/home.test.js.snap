// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches the snapshot" 1`] = `
<div>
  <p>
    Hello World
  </p>
  <input
    onChange={[Function]}
    value=""
  />
  <button
    onClick={[Function]}
  >
    Create Todo
  </button>
  <ul>
    <li>
      <span
        className="sc-bdVaJa gjWVUP"
        onClick={[Function]}
      >
        Hello Next.js
      </span>
    </li>
    <li>
      <span
        className="sc-bdVaJa gjWVUP"
        onClick={[Function]}
      >
        Learn Next.js is awesome
      </span>
    </li>
    <li>
      <span
        className="sc-bdVaJa gjWVUP"
        onClick={[Function]}
      >
        Deploy Indexs with Zeit
      </span>
    </li>
  </ul>
</div>
`;
